pandas/core/generic.py

```diff
@@ -3563,6 +3563,42 @@ def head(self, n=5):
         -------
         obj_head : type of caller
             The first n rows of the caller object.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({
+        ...     'col1' : ['A', 'A', 'B', np.nan, 'D', 'C', 'E', 'E', 'F'],
```

Comment:
Remove extra space after you finish the string:
```
'col1': 

Instead of:
'col1' :
```

Reply:
and being PEP8 compliant!